### DISCUSSION TITLE
看港澳台番剧需要设置什么吗？

### DISCUSSION THREAD

**Discussion by djjznba** [ywmoyue/biliuwp-lite#467] [Q&A] [answered]
显示地区不支持
![屏幕截图 2024-01-01 203429](https://github.com/ywmoyue/biliuwp-lite/assets/126856481/6cba43cd-d8a8-4b2c-b007-65008dc8188c)

**Comment by GD-Slime** [comment]
~查看 `设置 -> 代理` 中的相关说明。需要填写一些代理服务器地址。~
emmmm. 好像是出现了一些bug。我找一下问题原因吧
实测确实是有问题。已提交pr修复

**Comment by ywmoyue** [comment, marked as answer]
试试这个版本 https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.6.25.1

**Reply by djjznba** [reply]
非常感谢，可以使用
